renku/cli/_options.py

```diff
@@ -112,7 +112,10 @@ def check_siblings(graph, outputs):
 
     missing = siblings - outputs
     if missing:
-        msg = 'Include the files above or use --with-siblings option.'
+        msg = (
+            'Include the files above in the command '
+            'or use --with-siblings option.'
```

Comment:
Phrasing -- either

    Include the files above in the command or use --with-siblings.

or

        Include the files above in the command or use the --with-siblings option.